packages/node_modules/@webex/widget-recents/src/enhancers/setup.js

```diff
@@ -128,12 +130,14 @@ function getInitialSpaces(props) {
     widgetStatus
   } = props;
 
+  const spaceLoadCount = props.spaceLoadCount || DEFAULT_SPACE_COUNT;
```

Comment:
```suggestion
  const {spaceLoadCount = DEFAULT_SPACE_COUNT} = props;
```

Reply:
I was going to suggest this, but I couldn't remember if this handles updates/new props?